projects/demo/src/modules/components/input-time/index.html

```diff
@@ -158,6 +159,14 @@ <h3>DI-tokens for input-configurations:</h3>
             >
                 Items to choose
             </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="itemsHidden"
+                documentationPropertyType="boolean"
+                [(documentationPropertyValue)]="itemsHidden"
+            >
+                Disable the display of the dropdown list
```

Comment:
Should we just use `inject(TuiDropdownOpen, {optional: true}).tuiDropdownEnabled` instead of creating new prop ? 🤔

Reply:
I'm not sure `tuiDropdownEnabled` exists for legacy controls, this prop will only be needed in legacy, we would definitely implement this differently once we rewrite `InputTime` to new `Textfield`